Src/Add-AzureDtlVM.ps1

```diff
@@ -288,7 +288,7 @@ try {
     [array] $failed = $vms | ? { $_.Properties.provisioningState -eq 'Failed' }
     LogOutput "Detected $($failed.Count) failed VMs"
 
-    RemoveBatchVms -vms $failed -batchSize $removeBatchSize -credentialsKind $credentialsKind
+    RemoveBatchVms -vms $failed -batchSize $removeBatchSize -credentialsKind $credentialsKind -profilePath $profilePath
```

Comment:
Have you tested that it still works after passing the parameter in? Who removed in the first place and why?